### PR TITLE
Add places to nata333x2 san diego file

### LINKED ISSUE #11: Create a file for San Diego Recommendations
# San Diego (Poway), CA

- [ ] Create a branch
- [ ] Add commit(s)
- [ ] Open a pull request
- [ ] Have a discussion
- [ ] Seek approval
- [ ] Merge pull request

### PR DESCRIPTION
**Pull request by kgalliher**
closes #11 🐙

### CODE REVIEW

File: natas333x2-sandiego.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# San Diego, CA
+
+### Places to eat
+One of 500 __"bertos".  Albertos, Aibertos, Jillbertos, 
+
+### Places to visit
+Rattlesnake Creek hiking trail.
+
```
Comment (ptsushima) [inline natas333x2-sandiego.md]: How long is this hike?

**Comment by ptsushima** [comment]
🐶

File: natas333x2-sandiego.md (hunk 1 above)
Comment (kgalliher) [reply natas333x2-sandiego.md]: 2.2 miles.  It's in a canyon with a visitor center at the middle of the loop.

File: natas333x2-sandiego.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+### Places to Avoid
+East County
```
Comment (githubteacher) [inline natas333x2-sandiego.md]: 😭

File: natas333x2-sandiego.md (hunk 1 above)
Comment (githubteacher) [reply natas333x2-sandiego.md]: Where is this? Mission Gorge?

**Review by ptsushima — COMMENTED** [review]
That was a nice hike! 🚶

**Pull request merged**